### PR TITLE
Overview Section Added

### LINKED ISSUE #11: Overview section
#### Description
Update overview section in `README.md`

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#12
close #11  
#### What does this implement/fix? Explain your changes.
- overview section added to `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
---
+Dataset includes Direct Borohydride Fuel Cell (DBFC) impedance and polarization test in anode with Pd/C, Pt/C and Acta catalysts. In fact, different concentration of Sodium Borohydride (SBH), applied voltages and various anode catalysts loading with explanation of experimental details of electrochemical analysis are considered in data. Voltage, power density and resistance of DBFC change as a function of weight percent of SBH (%), applied voltage and amount of anode catalyst loading that are evaluated by polarization and impedance curves with using appropriate equivalent circuit of fuel cell. Can be stated that interpretation of electrochemical behavior changes by the data of related cell is inevitable, which can be useful in simulation, power source investigation and depth analysis in DB fuel cell researches.
```
Comment (sepandhaghighi) [inline README.md]: This dataset includes ...

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sepandhaghighi) [inline README.md]: Replace `Acta` by `Pd decorated Ni–Co/rGO`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
---
+This Dataset includes Direct Borohydride Fuel Cell (DBFC) impedance and polarization test in anode with Pd/C, Pt/C and Acta catalysts. In fact, different concentration of Sodium Borohydride (SBH), applied voltages and various anode catalysts loading with explanation of experimental details of electrochemical analysis are considered in data. Voltage, power density and resistance of DBFC change as a function of weight percent of SBH (%), applied voltage and amount of anode catalyst loading that are evaluated by polarization and impedance curves with using appropriate equivalent circuit of fuel cell. Can be stated that interpretation of electrochemical behavior changes by the data of related cell is inevitable, which can be useful in simulation, power source investigation and depth analysis in DB fuel cell researches.
```
Comment (sepandhaghighi) [inline README.md]: 1. This dataset (**lowercase**) ...
2. Replace `Acta` by `Pd decorated Ni–Co/rGO`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sadrasabouri) [reply README.md]: Oh right. :open_mouth:

**Review by kasraaskari — APPROVED** [review]
Thanks a lot

**Review by sepandhaghighi — APPROVED** [review]

**Review by sarminh — APPROVED** [review]
Fine and approved.

**Pull request merged**